### DISCUSSION TITLE
Tide (fish shell prompt plugin)

### DISCUSSION THREAD

**Discussion by ravener** [catppuccin/catppuccin#2217] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Most components have a bg_color variable that can be colored https://github.com/IlanCosman/tide/wiki/Configuration

You can see the default colors here https://github.com/IlanCosman/tide/tree/main/functions/tide/configure/configs

I'm not exactly sure how to go by this, and when I tried to swap some catppuccin colors onto some of them the text contrast was bad, hopefully there is a way to change the text color but I couldn't find it.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by jocelynthode** [comment]
Hey, I tried to quickly implement something that uses catppuccin. I have not tested it thoroughly.

https://github.com/jocelynthode/catppuccin-tide

**Reply by qianbinbin** [reply]
I tried `catppuccin_tide macchiato rainbow` but it looks something wrong with the colors

<img width="492" height="82" alt="截屏2026-04-30 03 09 27" src="https://github.com/user-attachments/assets/5e4cadd4-92c7-47b0-86cb-73b835dc4161" />
